lib/fastlane/plugin/firebase_test_lab/helper/ios_validator.rb

```diff
@@ -21,7 +21,17 @@ def self.validate_ios_app(file_path)
             unless size == 1
               UI.user_error!("The app bundle may contain only one scheme, #{size} found")
             end
-            _, scheme_conf = conf.first
+            
+            scheme_conf = nil
+            conf.each_key { |key| 
```

Comment:
We could use conf.each since you need the value as well (rather than conf[key]).

Reply:
Good suggestion. I've updated the commit.